api/apps/api/src/migrations/api/1650984916463-AddsExportIdToPublicProject.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddsExportIdToPublicProject1650984916463
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+          ALTER TABLE published_projects
+            ADD COLUMN export_id varchar;
```

Comment:
I don't think so, the `published-project` entity in the end is a copy of the actual `project`. I made this question as well regarding the `scenarioId` used for the preview map generation a while ago, and in the end linking this kind of copycat entity wasn't needed.

In fact, when you clone the public-project you are actually copying a `project` so I think it will be a little bit overkill if we link those two tables, but maybe @hotzevzl or @aagm have strong opinions about this here.